.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Red-DiscordBot Support
```

I think it's gonna fit better to other position on templates list ("Bug report", "Feature request", etc.)if we will call this "Support question" 
```suggestion
  - name: Support question
```